app/services/github_service.rb

```diff
@@ -0,0 +1,13 @@
+class GithubService
+  def conn
+    Faraday.new(url: 'https://api.github.com') do |faraday|
+      # faraday.headers['Authorization'] = ENV['GITHUB_API_KEY']
+      faraday.headers['Authorization'] = "token 484cabdee3b4e5de4226ed80470600b274e435d9"
+    end
```

Comment:
Travis CI fails if the token is hidden

Reply:
![](https://media3.giphy.com/media/XmpZQTWqvkBTa/giphy.gif?cid=5a38a5a26cb3640e55435979b64c51691b0af819e4be0081&rid=giphy.gif)